README.md

```diff
@@ -340,32 +340,33 @@ url("data/test.php?foo=bar");
 ```
 
 
-### Load tests in an iframe ###
+### Run tests in an iframe ###
 
-Loads a given page constructing a url with fileName: `"./data/" + fileName + ".html"`
-and fires the given callback on jQuery ready (using the jQuery loading from that page)
-and passes the iFrame's jQuery to the callback.
+Some tests may require a document other than the standard test fixture, and
+these can be run in a separate iframe. The actual test code and assertions
+remain in jQuery's main test files; only the minimal test fixture markup
+and setup code should be placed in the iframe file.
 
 ```js
-testIframe( fileName, testName, callback );
```

Comment:
Possibly? I don't think it will be nicely encapsulated because the iframe test does not include QUnit.